custom/icds_reports/utils/aggregation_helpers/distributed/ccs_record_monthly.py

```diff
@@ -295,7 +295,4 @@ def aggregation_query(self):
     def indexes(self):
         return [
             'CREATE INDEX IF NOT EXISTS crm_awc_case_idx ON "{}" (awc_id, case_id)'.format(self.tablename),
-            'CREATE INDEX IF NOT EXISTS crm_person_add_case_idx ON "{}" (person_case_id, add, case_id)'.format(
-                self.tablename
-            )
```

Comment:
I believe this was originally added for https://github.com/dimagi/commcare-hq/blob/3d5f09c80e2ccc0caac3361b36b05d2cfae086ff/custom/icds_reports/utils/aggregation_helpers/distributed/mbt.py#L246-L248

However I think that `(supervisor_id, month)` is likely a much better index because it ensures that only one month's data is included in the join, and only the correct supervisors (of which we will need all supervisors in the state)